ocaml/xapi/test_vdi_cbt.ml

```diff
@@ -0,0 +1,69 @@
+
+let register_smapiv2_server (module S: Storage_interface.Server_impl with type context = unit) sr_ref =
+  let module S = Storage_interface.Server(S) in
+  let rpc = S.process () in
+  let dummy_query_result = Storage_interface.({ driver=""; name=""; description=""; vendor=""; copyright=""; version=""; required_api_version=""; features=[]; configuration=[]; required_cluster_stack=[] }) in
+  Storage_mux.register sr_ref rpc "" dummy_query_result
```

Comment:
We can move this out into a `test_storage_common.ml` helper, when we start to test other storage-related XenAPI calls this way.